docker-images/dockerBuild.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+########################
+# Purpose: download the required docker images for installation
+# Author: Prasad Talasila
+# Date: 16-April-2017
+# Invocation: $sudo bash docker_pull.sh
+# Dependencies: This script requires docker to be already installed on the system.
+########################
+
+sudo bash ./docker-pull.sh
+cd ../main_server/ || exit
+sudo docker build -t main_server .
```

Comment:
It might be better to also log the output for future reference. We can do that by modifying the command to
```
sudo docker build -t main_server . | tee ../main_server/main_server_container_build.log
```

Reply:
please modify the other docker build commands suitably